jobs/influxdb/templates/influxd_ctl.erb

```diff
@@ -78,16 +78,18 @@ daemon=/var/vcap/packages/influxdb/influxd
 bootstrapper=/var/vcap/packages/bootstrapper/bootstrapper
 
 # pid file for the daemon
-pidfile=/var/vcap/store/influxdb/influxd.pid
+pidfile=/var/vcap/sys/run/influxdb/influxd.pid
 
 # Configuration file
 config=/var/vcap/jobs/influxdb/config/influxdb.conf
 
-mkdir -p /var/vcap/store/influxdb
+mkdir -p $(basename $pidfile)
```

Comment:
Haha, yes. I misunderstood what you were trying to say earlier

On Thu, Sep 8, 2016, 10:52 AM Alex Suraci <notifications@github.com> wrote:

> In jobs/influxdb/templates/influxd_ctl.erb
> <https://github.com/vito/influxdb-boshrelease/pull/9#discussion_r78022044>
> :
>
> >
> >  # Configuration file
> >  config=/var/vcap/jobs/influxdb/config/influxdb.conf
> >
> > -mkdir -p /var/vcap/store/influxdb
> > +mkdir -p $(basename $pidfile)
>
> Well, it'd end up running mkdir -p influxd.pid which is a bit borked. :)
>
> /t/deployments (master) $ basename /var/vcap/sys/run/influxdb/influxd.pid
> influxd.pid
> /t/deployments (master) $ dirname /var/vcap/sys/run/influxdb/influxd.pid
> /var/vcap/sys/run/influxdb
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vito/influxdb-boshrelease/pull/9/files/a8ce518651ee9689bf6d644ffaca0b112499b8dd#r78022044>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASKlZnEZrCWnN2zSTWWBOlM5X_D5ZLxks5qoCETgaJpZM4J3EOA>
> .
>